h/static/scripts/group-forms/components/ModerationStatusSelect.tsx

```diff
@@ -0,0 +1,60 @@
+import type { IconComponent } from '@hypothesis/frontend-shared';
+import { FilterIcon } from '@hypothesis/frontend-shared';
+import { CautionIcon } from '@hypothesis/frontend-shared';
+import { RestrictedIcon } from '@hypothesis/frontend-shared';
+import { CheckAllIcon } from '@hypothesis/frontend-shared';
+import { DottedCircleIcon, Select } from '@hypothesis/frontend-shared';
```

Comment:
The non-type imports can be combined.

Reply:
My IDE is a bit stupid 😂. This should solve the issue https://github.com/hypothesis/h/pull/9550